### PR TITLE
release/v2.1.0 (develop)

### PR DESCRIPTION
**Pull request by AlbertPG**

### CODE REVIEW

File: app/src/main/java/org/pesmypetcare/mypetcare/services/googlecalendar/GoogleCalendarAdapter.java [hunk 1]
```diff
@@ -64,6 +64,7 @@ public void deleteEvent(Pet pet, Event event) {
     @Override
     public void newSecondaryCalendar(Pet pet) {
         ExecutorService executorService = Executors.newSingleThreadExecutor();
+        System.out.println("CALENDAR TOKEN "+ pet.getOwner().getGoogleCalendarToken());
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/services/googlecalendar/GoogleCalendarAdapter.java:67]: Codacy Issue found: ['+' is not preceded with whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5650467)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5650467&bid=18501237)

File: app/src/main/java/org/pesmypetcare/mypetcare/services/pet/PetManagerAdapter.java [hunk 2]
```diff
@@ -102,6 +102,22 @@ public boolean registerNewPet(User user, Pet pet) {
             }
         });
 
+        executorService.shutdown();
+        try {
+            executorService.awaitTermination(5, TimeUnit.MINUTES);
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/services/pet/PetManagerAdapter.java:107]: Codacy Issue found: ['5' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5650467)

**Review by enrichp — DISMISSED** [review]

**Review by santidrj — APPROVED** [review]
:+1:

**Review by enrichp — APPROVED** [review]

**Pull request merged**